app/gateway/2.8.x/plan-and-deploy/security/secrets-management/backends/env.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Environment Variable Vault
+beta: true
+badge: free
+---
+
+## Configuration
+
+Storing secrets in Environment Variables is a common way as they can be injected at build time.
+There is no prior configuration needed.
+
+## Examples
+
+Define a secret in a environment variable:
```

Comment:
We could also point that variable can be `json` string if you want to configure multiple secrets in single environment variable:

```
export PG_CREDS='{"username":"user", "password":"pass"}'
```

Then you can reference them with:
```
{vault://env/pg-creds/username}
{vault://env/pg-creds/password}
```